app/views/admin/dashboard/show.html.erb

```diff
@@ -11,11 +11,11 @@
     <th>Actions</th>
   </tr>
   <% @facade.tutorials.each do |tutorial| %>
-    <tr class="admin-tutorial-card">
+    <tr class="admin-tutorial-card-<%= tutorial.id %>">
       <td><%= link_to tutorial.title, tutorial_path(tutorial) %></td>
       <td>
-        <%= link_to "Edit", edit_admin_tutorial_path(tutorial) %> |
-        <%= link_to 'Destroy', tutorial_path(tutorial),
+        <%= button_to "Edit", edit_admin_tutorial_path(tutorial), method: :get %>
+        <%= button_to 'Destroy', admin_tutorial_path(tutorial),
```

Comment:
This was such a weird error. I wonder why we weren't able to override the method for the `link_to`